src/_tutorials/server/cmdline.md

```diff
@@ -96,7 +95,7 @@ brief description of key app features, click the highlighted code below.
 
 You might notice that dcat depends on a package named **args**.
 To get the args package, use the
-[pub package manager](/guides/packages).
+[dart pub package manager](/guides/packages).
```

Comment:
It's still the pub package manager; only the command needs the `dart` preface.

```suggestion
[pub package manager](/guides/packages).
```

Reply:
Okay, thanks for the clarification.